### DISCUSSION TITLE
Multiple *.html presentations in reveal home directory

### DISCUSSION THREAD

**Discussion by matiasgrandi** [hakimel/reveal.js#3485] [Q&A]
Hi!
I've have made several presentations with revealjs. All of them with the same template, css, etc.

I've have named the files as:
-  index.html: for presentation about "cars"
- index1.html: for presentation about "boats"
- index2.html: for presentation about "planes"

The only one working nice is the first one, about "cars". When I point with the browser to /myslidesfolder/index1.html, it loads without rendering ccs styles. Same with index2.html.

Do I have to modify something in the code of each html?

Thanks!

**Comment by libralibra** [comment]
yes, you do! you need to manually change the reference of all .css, .js, and other related files in the html file. I've done something similar to keep multiple html files in different subfolders and they worked fine.

**Comment by t-fritsch** [comment]
hey,
as long as the html files are in the same folder, you shouldn't have to do any modifications to assets paths (css, js, images, etc.). ie. if it works for `index.html`, it should work too for `index1.html` and `index2.html`.

But if index1 and index2 are in a different subfolder, you then should adapt path of every asset used in the html file as stated by @libralibra.
